examples/oneapi/cpp/source/kmeans/kmeans_dense_batch.cpp

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
```

Comment:
Rename example `kmeans_dense_batch.cpp` -> `kmeans_lloyd_dense_batch.cpp`. Algorithm - [Type of task (optional)] - Method - Input format - Mode